src/promptflow-rag/pyproject.toml

```diff
@@ -32,7 +32,7 @@ packages = [
 
 # dependencies
 [tool.poetry.dependencies]
-python = "<4.0,>=3.8"
+python = "<4.0,>=3.8.1"
```

Comment:
Why we change here?

Reply:
I can undo this, since it's not related to the task. But I was noticing version problems when attempting to setup the repo with poetry. This fixed them.